_includes/releases/v23.1/v23.1.0-alpha.8.md

```diff
@@ -0,0 +1,139 @@
+## v23.1.0-alpha.8
+
+Release Date: March 27, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-0-alpha-8-security-updates">Security updates</h3>
+
+- Previously, users could gain unauthorized access to [statement diagnostic bundles](../v23.1/ui-statements-page.html#diagnostics) they did not create if they requested the bundle through an HTTP request to `/_admin/v1/stmtbundle/<id>` and correctly guessed its (non-secret) `ID`.  This change now locks down the endpoint behind the SQL gating and uses the correct SQL user in the HTTP session as identified by their cookie. [#99051][#99051]
```

Comment:
I would express this a bit differently. The way this is written prompts the question -- did we have any data breaches?

Something like, "This change ensures that only authorized SQL users can request statement diagnostic bundles."